src/util.rs

```diff
@@ -145,14 +139,10 @@ pub async fn hash_all_archives(
         let path = path.clone();
         pool.spawn_blocking(move || -> Result<String> {
             let mut m = Digest::new();
+            debug!("start hash file {:?}", path);
             let reader = File::open(&path)
                 .with_context(|| format!("Failed to open file for hashing: {:?}", path))?;
-            let mut archive = Archive::new(reader);
-            while let Some(entry) = archive.next_entry() {
-                let entry = entry?;
```

Comment:
I believe this could lead to increasingly delicate and intricate behavior. I suggest removing this logic entirely unless someone wishes to provide a correct implementation.

Reply:
What are the consequences of removing it entirely? The timestamps are still present so on mac os we get (according to the comment) no cache hits for `ar` files at all. How many `ar`-format files are present in an average compilation run C/C++/rust?

Reply:
Hi, I add a TODO notes here for hash_all_archives which explains what happened here. PTAL.